baseapp/abci_utils.go

```diff
@@ -368,7 +369,7 @@ func (ts *defaultTxSelector) Clear() {
 }
 
 func (ts *defaultTxSelector) SelectTxForProposal(_ context.Context, maxTxBytes, maxBlockGas uint64, memTx sdk.Tx, txBz []byte) bool {
-	txSize := uint64(len(txBz))
+	txSize := uint64(tmtypes.ComputeProtoSizeForTxs([]tmtypes.Tx{txBz}))
```

Comment:
Can you add a test?

Reply:
[TestDefaultProposalHandler_NoOpMempoolTxSelection](https://github.com/cosmos/cosmos-sdk/blob/main/baseapp/abci_utils_test.go#L285)
Can this test be used

Reply:
Sure, adding a case there works!